swagger/medco-connector.yml

```diff
@@ -103,6 +103,20 @@ paths:
       security:
         - medco-jwt: ["medco-explore"]
 
+  /node/status:
+    get:
+      tags:
+        - "medco-node"
+      summary: "Get info about node status."
+      operationId: "getStatus"
+      responses:
+        200:
+          $ref: "#/responses/statusResponse"
+        default:
+          $ref: "#/responses/errorResponse"
+      security:
+        - medco-jwt: ["medco-genomic-annotations"]
```

Comment:
As said in comment: agreed for separate role.

Reply:
Agreed